api/src/test/java/org/openmrs/validator/VisitValidatorTest.java

```diff
@@ -441,11 +441,44 @@ public void validate_shouldPassValidationIfFieldLengthsAreCorrect() {
 	@Test
 	public void validate_shouldFailValidationIfFieldLengthsAreNotCorrect() {
 		Visit visit = makeVisit(42);
-		visit
-		        .setVoidReason("too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text");
-		
+		visit.setVoidReason("too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text");
```

Comment:
@tenas97 was this ticket simply about adding unit tests? Or even changing the implementation to ensure that the visit start date can't be before the birth date of the patient?